web/src/ui_init.js

```diff
@@ -328,12 +328,12 @@ export function initialize_kitchen_sink_stuff() {
         message_unhover();
     });
 
-    $("#main_div").on("mouseover", ".sender_info_hover", function () {
+    $("#main_div").on("mouseover", ".sender_info_hover span > *", function () {
```

Comment:
While this method might work, we need to keep in mind that if we add another element to the span container later on, it might cause us to have to fix the hover effect all over again. Instead, maybe we could consider explicitly specifying the containers we want the hover effect to apply to. What do you think?

Reply:
Another approach I was thinking that maybe we could try using `inline-flex` instead of `flex` for our `span` container. This would make it content width and help us avoid problems like the one we're currently experiencing. However, we'd need to test it out to make sure it works.

Reply:
@shameondev I think explicitly specifying the containers we want the hover effect to apply to is a better approach like this: 
`$("#main_div").on("mouseover", ".inline_profile_picture, .sender_name", function () {`
If you approve the selectors above I will push the updated PR with the changes.

I don't think the other approach you specified of using inline-flex instead of flex will work as it will have the same issue you mentioned: 

>we need to keep in mind that if we add another element to the span container later on, it might cause us to have to fix the hover effect all over again

Please provide your feedback on the above.

Reply:
Awesome! That's something I've been thinking about! You can definitely go ahead and push this code.